packages/core/src/node/backend-application.ts

```diff
@@ -17,6 +20,23 @@ export interface BackendApplicationContribution {
     onStart?(server: http.Server): void;
 }
 
+@injectable()
+export class BackendApplicationCliContribution implements CliContribution {
+
+    port: number;
+    hostname: string | undefined;
```

Comment:
Did you intentionally leave these implicit `public`?

Reply:
Yes, those properties are being accessed from outside.